setup.py

```diff
@@ -47,7 +47,7 @@
     'docker>=4.1.0,<4.2.0',
     'Flask-BabelEx>=0.9.4',
     'pipenv==2018.11.26',
-    'PyYAML>=5.1.2,<5.2.0',
+    'PyYAML>=5.1.2',
```

Comment:
I am thinking right now we should make packages less than the major version for third-party packages:

- follows semantic versioning, so if everybody plays by those rules, we are good
- if third-party packages don't play by those rules, the instance lock file should save us from their breaking changes

So what about something like:

`'PyYAML>=5.1.2,<6.0.0'` (or `'PyYAML~=5.1.2'`)

Reply:
From my perspective for external packages is better to leave them open. If we pin, we will be stuck forever in a version, because being realistic we do not have the manpower to be checking versions that come out of every library. And since the semantic versioning does not fully apply, we might miss a security patch  (what if a security patch is a breaking change?). From my perspective it's better to get at some point a breaking change rather than not get a security patch.

If something gets in place to follow us and use things that follow semantic versioning I'm happy to do so. But from my point of view it's and Invenio decision (like, lets have a talk with the architects) rather than here in a single module.

I know we did it in cookiecutter, but its for our modules during release since when we are stable we will depend on bundles.

Reply:
Let's videochat about it tomorrow or today.